src/main/java/frc/robot/Constants.java

```diff
@@ -92,6 +92,7 @@ public static final class ArmConstants {
 
     public static final double kHomePosition = 0.0;
     public static final double k1stRowPosition = 0.0;
+    public static final double kOffFloorPosition = 0.0;
     public static final double k2ndRowPosition = 0.0;
     public static final double k3rdRowPosition = 0.0;
```

Comment:
Maybe rename these at some point to indicate position units (motor revolutions, aka cycles, right?).